<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoorConfigDialog</class>
 <widget class="QDialog" name="DoorConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="6" column="1" rowspan="3">
    <widget class="QGroupBox" name="GroupBox_Preview">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGraphicsView" name="GraphicsView_Preview"/>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="6">
    <widget class="QGroupBox" name="GroupBox_EntitySet">
     <property name="title">
      <string>EntitySet</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="Label_EntitySetID">
        <property name="text">
         <string>EntitySet ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="ComboBox_EntityID"/>
      </item>
      <item>
       <widget class="QLabel" name="Label_EntityFilter">
        <property name="text">
         <string>Entity Filter:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="TableView_EntityFilter">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="TextEdit_AllTheEntities">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="GroupBox_PositionAndSize">
     <property name="title">
      <string>Position and size</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QSpinBox" name="SpinBox_DoorX"/>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="Label_DoorHeight">
          <property name="text">
           <string>Height</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="Label_DoorWidth">
          <property name="text">
           <string>Width</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="SpinBox_DoorY"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="Label_x1">
          <property name="text">
           <string>X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="Label_y1">
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSpinBox" name="SpinBox_DoorWidth">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="SpinBox_DoorHeight">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" rowspan="2">
    <widget class="QGroupBox" name="GroupBox_DestinationDoor">
     <property name="title">
      <string>Destination Door:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="GraphicsView_DestinationDoor">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QGroupBox" name="GroupBox_AppearancePlace">
     <property name="title">
      <string>The place of Wario's appearance</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="Label_WarioX">
          <property name="text">
           <string>X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="Label_WarioY">
          <property name="text">
           <string>Y</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="SpinBox_WarioX"/>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="SpinBox_WarioY"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QComboBox" name="ComboBox_DoorDestinationPicker"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="Label_DoorType">
     <property name="text">
      <string>Door Type:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="Label_DoorDestination">
     <property name="text">
      <string>Door Destination:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QComboBox" name="ComboBox_DoorType"/>
   </item>
   <item row="8" column="0">
    <widget class="QGroupBox" name="GroupBox_BGM_ID">
     <property name="title">
      <string>BGM ID:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QSpinBox" name="SpinBox_BGM_ID"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DoorConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DoorConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
